docs/6.x/changelog.md

```diff
@@ -37,6 +37,14 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 6.x Releases
 
+### 6.1.12 LTS (November 11th, 2019)
+
+#### Improvements
+
+* Preflight checks are now executed during join operation ([#853](https://github.com/gravitational/gravity/pull/853)).
```

Comment:
nit: `Execute preflight checks during join operation` to keep the same sentence form?